tests/Core/Tokenizer/ScopeOwnerTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Tests that scope owner (scope_condition, conditions) is set correctly
+ *
+ * @author  Dan Wallis <dan@wallis.nz>
+ * @license https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Tests\Core\Tokenizer;
+
+use PHP_CodeSniffer\Tests\Core\AbstractMethodUnitTest;
+
+final class ScopeOwnerTest extends AbstractMethodUnitTest
```

Comment:
Could you please change this to use the Tokenizer specific testcase I introduced in #314 ? That should allow code coverage to be recorded for this.